Part1/code/main.py

```diff
@@ -1,47 +1,42 @@
 from VisualAcuityDataset import VisualAcuityDataset
 from LimitedColorPerception import LimitedColorPerceptionDataset
+from torch.utils.data import DataLoader
 import numpy as np
 import matplotlib.pyplot as plt
-from torchvision import transforms
 from torchvision.datasets import STL10
 
-dataset = STL10('./data', split='train', download=True)
-# Create the VisualAcuityDataset with a specific month_age
-month_ages = list(range(0, 13))  # Create a list of ages from 0 to 12
-va_datasets = []
-lc_datasets=[]
-for month_age in month_ages:
-    va_datasets.append(VisualAcuityDataset(dataset, month_age=month_age))
-    lc_datasets.append(LimitedColorPerceptionDataset(dataset, month_age=month_age))
+# STL10 dataset by Stanford University (https://cs.stanford.edu/~acoates/stl10/)
```

Comment:
the origin is maybe true, but the place we get the dataset is torchvision.datasets, not cs.stanford.edu

Reply:
Torch is the environment who provides the dataset which we are not interested in. However, the dataset's owner and provider is still Stanford University.